msg.go

```diff
@@ -413,12 +413,6 @@ Loop:
 		case 0x00:
 			if c == 0x00 {
 				// end of name
-				if len(s) == 0 {
-					if ptr == 0 {
-						off1 = off
-					}
-					return ".", off1, nil
-				}
```

Comment:
Yeah, this is a rebase artifact. The commit message acknowledges that this was independently fixed.